lib/datadog/core/configuration/components.rb

```diff
@@ -51,6 +52,15 @@ def build_runtime_metrics_worker(settings)
             Core::Workers::RuntimeMetrics.new(options)
           end
 
+          def build_telemetry(settings)
+            if @telemetry
```

Comment:
Leave a comment here saying we resuing a previous instance of telemetry in the "if true" case.